site/content/tutorial/06-bindings/07-multiple-select-bindings/text.md

```diff
@@ -18,4 +18,4 @@ Returning to our [earlier ice cream example](tutorial/group-inputs), we can repl
 </select>
 ```
 
-> Press and hold the `control` key for selecting multiple options.
+> Press and hold the `control` key (on Windows) or the `command` key (on MacOS) for selecting multiple options.
```

Comment:
What about Linux? Personally I think if you use MacOS you should know that the key is different already, but if it must be spelled out we could at least do it such that it's accurate for other OSes as well:

```suggestion
> Press and hold the `control` key (or the `command` key on MacOS) for selecting multiple options.
```

Reply:
Oh, yes, that would be better. Thanks for pointing that out.